### PR TITLE
chore: improve npm scripts

### PR DESCRIPTION
**Pull request by ghost**
The origin `--fix` is directly added after `lint`, but according
to @popomore's idea and other references, it seems we should put it
after `test`.

/cc：@popomore, @dead-horse

### CODE REVIEW

**Review by popomore — APPROVED** [review]

**Comment by popomore** [comment]
Can you follow the commit format https://github.com/eggjs/egg/blob/master/CONTRIBUTING.md#commit-message-format

**Comment by Maledong** [comment]
@popomore：Fixed :)

File: package.json [hunk 1]
```diff
@@ -53,10 +53,10 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "test": "npm run lint && npm run test-local",
+    "lint": "eslint app config lib test *.js",
```
Comment (atian25) [inline package.json]: `eslint .`

**Review by atian25 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json]: Fixed :)

**Pull request merged**